app/webpacker/src/javascripts/commodities.js

```diff
@@ -798,7 +798,10 @@
                         data: obj
                       }
                     }]);
-                    $('.js-commodity-picker-select').parents('form:first').trigger("submit");
+
```

Comment:
Do we want to write a test for this change?

Reply:
In principle yes, but the existing spec is skeletal at best. I'll need write a more complete spec for existing behaviour in order to add a spec for this small change. I can look at it this afternoon